modules/ossm-directing-egress-traffic-through-a-gateway-kubernetes-gateway-api-ambient-mode.adoc

```diff
@@ -0,0 +1,154 @@
+// This procedure is used in the following assembly:
+// * service-mesh-docs-main/gateways/ossm-directing-outbound-traffic-through-a-gateway
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-directing-egress-traffic-through-gateway-kubernetes-gateway-api-ambient-mode_{context}"]
+= Directing egress traffic through a gateway using the {k8s} Gateway API in ambient mode
+
+Use the {k8s} Gateway API and waypoint proxy to direct outbound HTTP traffic through an egress gateway.
+
+.Prerequisites
+
+* You have installed the {SMProduct} Operator version 3.2 or later.
+
+* You configured the `Istio` and `IstioCNI` resources with ambient profile.
+
+* You have created a `Ztunnel` resource.
+
+.Procedure
+
+. Optional: Enable the {k8} Gateway API custom resource definitions (CRDs). 
++
+[NOTE]
+====
+As of {k8s} 1.28 and {ocp-product-title} 4.18 or earlier version of {product-title}, the {k8s} Gateway API CRDs are not available by default and you must enable the CRDs before you can use them. {ocp-product-title} 4.19 and later versions enable the CRDs by default.
+====
+
+. Create a namespace called `egress-gateway` by running the following command:
++
+[source,terminal]
+----
+$ oc create namespace egress-gateway
+----
+
+. Apply the ambient mode label to the namespace by running the following command:
++
+[source,terminal]
+----
+$ oc label namespace egress-gateway istio.io/dataplane-mode=ambient
+----
+
+. Create a YAML file named `egress-gateway-cr.yaml` that defines the egress gateway.
```

Comment:
We are creating a ServiceEntry and HTTPRoute in the yaml below and not the egress gateway. Please rephrase the sentence as follows.

```suggestion
. Create a YAML file named `se-httproute.yaml` that defines the ServiceEntry along with the HTTPRoute.
```